client/state/selectors/test/get-magic-login-requested-auth-successfully.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { getMagicLoginRequestedAuthSuccessfully } from '../';
+
+describe( 'getMagicLoginRequestedAuthSuccessfully()', () => {
+	it( 'should return false if there is no information yet', () => {
+		const status = getMagicLoginRequestedAuthSuccessfully( undefined );
+		expect( status ).to.be.false;
+	} );
+
+	it( 'should return true if true', () => {
+		const status = getMagicLoginRequestedAuthSuccessfully( {
+			login: {
+				magicLogin: {
+					requestAuthSuccess: true,
+				},
+			},
+		} );
+		expect( status ).to.be.true;
+	} );
+
+	it( 'should return false if false', () => {
```

Comment:
Would be better to describe is as something like `should return false if requested auth failed`